common/persistence/interface.KvDriver.php

```diff
@@ -34,12 +33,14 @@
     /**
      * Stores a value, implementing time to live and nx is optional
      * Should throw an exception if an option is not supported
-     * 
+     *
      * @param string $id
      * @param string $value
      * @param string $ttl time to live in seconds
      * @param bool $nx Only set the key if it does not already exist
      * @return bool
```

Comment:
What does the bool return value indicate if errors are now handled via exceptions?